### DISCUSSION TITLE
The algorithm of eigenvector centrality

### DISCUSSION THREAD

**Discussion by Moon0722** [gephi/gephi#2549] [Q&A]
Hi, I'm using Gephi to analyze my dataset, and I've also used Ucinet to analyze the same one. 
I found out that the result of Eigenvector Centrality computed by Gephi is different from the result computed by Ucinet.
I wanna know the algorithm of Eigenvector Centrality used by Gephi.
Could you please tell me where can I get this kind of information?
Thank you so much!

**Comment by ggGloria121** [comment]
Hi, I think maybe it's because the default number of iterations computed in gephi is 100? But have you found the algorithm for gephi to calculate the eigenvectors centrality? I also had this problem recently but it has not been resolved.
